api/main.go

```diff
@@ -35,3 +35,20 @@ func StartServer(reload bool) {
 
 func Shutdown() {
 }
+
+// make map string string
+func mmss(data ...string) map[string]string {
```

Comment:
This function does not belong in the main file

Reply:
Good suggestion